server/netty-server/src/main/scala/sttp/tapir/server/netty/NettyConfig.scala

```diff
@@ -73,6 +79,10 @@ object NettyConfig {
     shutdownEventLoopGroupOnClose = true,
     socketBacklog = 128,
     socketKeepAlive = true,
+    requestTimeout = FiniteDuration(60, SECONDS),
```

Comment:
**suggestion:** After importing `import scala.concurrent.duration._` you can use a bit nicer expressions like `requestTimeout = 60.seconds` etc.